articles/architecture-scenarios/_includes/_branding/_password-reset.md

```diff
@@ -1 +1,8 @@
 The [Password Reset](/universal-login/password-reset) page is used whenever a user takes advantage of password change functionality and, as with the login page, you can [customize it](/universal-login/password-reset#edit-the-password-reset-page) to reflect your organization's particular branding requirements. 
+
+<% if (platform === "b2b") { %>
+If your organization users will all be isolated from each other (as in, each organization gets its own Auth0 database connection), and you are [branding the universal login page by organization](#universal-login-and-login-pages) then it is important to make it clear on the change password page as well which organization the change password page is for. This can be done in a couple of ways:
```

Comment:
```suggestion
If users in your customer organizations will be isolated from each other (i.e. each organization gets its own Auth0 [Database Connection](/connections/database)), and you are branding the [universal login](#universal-login-and-login-pages) page by organization, then it's important to make sure you brand things like the [password reset](/universal-login/password-reset) page too - so that users know which organization they are targeting (e.g. for which organization the password change is occurring). This can be done in a couple of ways:```

Reply:
Added to PR 7818